unstructured_inference/inference/layoutelement.py

```diff
@@ -321,13 +327,72 @@ def get_elements_from_ocr_regions(ocr_regions: List[TextRegion]) -> List[LayoutE
             r.y1,
             r.x2,
             r.y2,
+            None,
             text=r.text,
             type="UncategorizedText",
         )
         for r in merged_regions
     ]
 
 
+def probably_contained(
+    element: Union[LayoutElement, Rectangle],
+    inside: Union[LayoutElement, Rectangle],
+    area_threshold: float = 0.9,
+) -> bool:
+    """This function checks if one element is inside other.
+        If is definetly inside returns True, in other case check
+    if the intersection is big enough."""
+    # Probably can be implemented usigng is_in, however this is based in area overlapping
+    if element.is_in(inside):
+        return True
+
+    intersected_area = element.intersection(inside)
+    if intersected_area:
+        return intersected_area.area >= inside.area * area_threshold
+    return False
+
+
+def separate(region_a: Union[LayoutElement, Rectangle], region_b: Union[LayoutElement, Rectangle]):
+    """Reduce leftmost rectangle to don't overlap with the other"""
+
+    def reduce(keep: Rectangle, reduce: Rectangle):
+        # Asume intersection
+
+        # Other is down
+        if reduce.y2 > keep.y2 and reduce.x1 < keep.x2:
+            # other is down-right
+            if reduce.x2 > keep.x2 and reduce.y2 > keep.y2:
+                reduce.x1 = keep.x2 * 1.01
+                reduce.y1 = keep.y2 * 1.01
```

Comment:
let's define constant values like `1.01` here somewhere that can be imported / changed.